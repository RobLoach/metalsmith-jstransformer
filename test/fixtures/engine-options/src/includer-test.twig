<h1>{{ pageTitle }}</h1>
{% include "@comps/included.html" %}
