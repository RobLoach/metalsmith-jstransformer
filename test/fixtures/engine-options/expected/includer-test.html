<h1>Twig Title!</h1>
<div class="included">This is included.html</div>
